lib/index.js

```diff
@@ -202,7 +229,14 @@ api.getPublicKey = (publicKey, actor, callback) => {
   if(actor !== null) {
     actor = actor || undefined;
   }
-  const publicKeyId = publicKey.id;
+
+  const {id: publicKeyId, owner, publicKeyPem} = publicKey;
+  // if `id` is not provided, both `owner` and `publicKeyPem` must be provided
+  if(!publicKeyId) {
+    assert.string(owner);
+    assert.string(publicKeyPem);
```

Comment:
Should assert/assert-plus be used for these sorts of checks?  If the code is run with NODE_NDEBUG set, then assert-plus calls are noops.  Just want to make sure if we try to optimize things and set that env var for an app that we still run important checks.

Reply:
it's using assert-plus

Reply:
Yes.  Is it ok to run this code in production with NODE_NDEBUG set?  That will disable all assert-plus asserts.

Reply:
Do you have documentation on NODE_NDEBUG?  I can't find anything: https://nodejs.org/docs/latest-v8.x/api/cli.html#

Provided that it doesn't get set if production mode is on or something, I think it's fine.

Reply:
It's maybe only an assert-plus feature:
https://github.com/joyent/node-assert-plus
The concern is if we shut it off for some other reason, we'll not realize it's disabling code here.
Could put some top level check like `if(process.env.NODE_NDEBUG) { throw ... }` just to be sure it's not set if we think this is an issue.

Reply:
@dlongley @davidlehn since assert-plus is finding its way into modules, should a check for this flag happen in `bedrock`, and maybe we just expose `assert` via bedrock.util?

Anyone think there is a real risk of this environment flag getting set unexpectedly?